bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md

```diff
@@ -0,0 +1,144 @@
+# Run a Rocket.Chat SDK Bot
+
+This bot is a simple example of how you can use [Rocket.Chat JS SDK](https://github.com/RocketChat/Rocket.Chat.js.SDK) methods directly. It
+does not handle errors, different messsage types, server resets and other production
```

Comment:
It is extremely basic (primitive/low level?) and does not handle.....

Or

Compared to other bots documented here it is extremely basic and does not handle blah